docs/output.md

```diff
@@ -223,6 +224,22 @@ Output Summaries:
 
 [Macrel](https://github.com/BigDataBiology/macrel) is a tool that mines antimicrobial peptides (AMPs) from (meta)genomes by predicting peptides from genomes (provided as contigs) and outputs all the predicted anti-microbial peptides found.
 
+### AMPcombi
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `ampcombi/`
+  - `ampcombi_complete_summary.csv`: summarized output from all AMP workflow tools (except hmmer_hmmsearch) in csv
+  - `ampcombi.log`: a log file generated by ampcombi
+  - `*_ampcombi.csv`: summarized output in csv for each sample
+  - `*_amp.faa*`: fasta file containing the amino acid sequences for all AMP hits for each sample
+  - `*_diamond_matches.txt*`: alignment file generated fby DIAMOND for each sample
+
+</details>
+
+[AMPcombi](https://github.com/Darcy220606/AMPcombi) summarizes parses the results of **antimicrobial peptide (AMP)** prediction tools (AMPIR, AMPLIFY, MACREL, HMMSEARCH and other non nf-core tools) into a single table and aligns the hits against a reference AMP database for functional and taxonomic classification.
```

Comment:
```suggestion
[AMPcombi](https://github.com/Darcy220606/AMPcombi) summarizes parses the results of **antimicrobial peptide (AMP)** prediction tools (AMPIR, AMPLIFY, MACREL, and other non nf-core tools) into a single table and aligns the hits against a reference AMP database for functional and taxonomic classification.
```

Reply:
Maybe also mention which columns it has (e.g. probabilitiesi, e-values etc.), this will help better orientate the user as to what sort of summary table this.